src/components/MultiSelect/MultiSelectMenuButton.tsx

```diff
@@ -112,6 +112,7 @@ const MultiSelectMenuButton = forwardRef<
             marginLeft="xs"
             name="close"
             size="xsmall"
+            title="Remove selected items"
```

Comment:
The entire span acts as a button and has an aria-label, but this SVG still needs its own title, right?

Reply:
Yes, but also no.  Rather than leaving the default title, which is not very descriptive, I added something in line with the actual functionality.  However, we have do have `aria-hidden="true"` applied, so the title does not get seen by assistive technology.